tests/net/bridge/prj.conf

```diff
@@ -5,8 +5,8 @@ CONFIG_NET_ETHERNET_BRIDGE=y
 CONFIG_NET_LOG=y
```

Comment:
Your commit message mentions `CONFIG_NET_IF_MAX_IPV6_COUNT` but you are not changing its value in conf file. Please fix the commit message.

Reply:
Hi @jukkar, thanks for correcting, I have addressed it.